src/tools/rbd_mirror/group_replayer/Replayer.cc

```diff
@@ -402,6 +386,7 @@ void Replayer<I>::handle_load_local_group_snapshots(int r) {
     return;
   }
 
+  bool is_restarted = false;
```

Comment:
Is this variable required? Could the `m_create_group_snaps` be repopulated with the latest values from `m_local_group_snaps` instead and then checked for entries?

PS: I am not in favour of having a separate `m_create_group_snaps` vector but this seems like it could be optimised.